src/components/categories-container/categories-container.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="categories-container">
```

Comment:
I have seen the component and the 4th and 5th categories are the same as the Figma design. They are a little big or large.
Can you review it?
![categories-view-1](https://github.com/FullHouseTeam/pet-paradise-client/assets/73414963/1ec96308-1c3a-4d00-bf3d-00e407d369ad)